### PR TITLE
Renamed the Microsoft Visual C++ Redistributable label from 2017 to 2019

### PR DESCRIPTION
**Pull request by DaisukeDaisuke**

### CODE REVIEW

**Comment by KygekDev** [comment]
@DaisukeDaisuke please read this:
https://github.com/DaisukeDaisukeTeam/pmmp.rtfd.io/commit/4ed53bfc158378cb66638bbb492e23b43185e39c#r58693312

**Comment by DaisukeDaisuke** [comment]
Thank you very much.
I will remove "2019".

**Comment by KygekDev** [comment]
> Thank you very much. I will remove "2019".

No, I mean you can remove the download link from this page: https://github.com/pmmp/pmmp.rtfd.io/pull/23/commits/e2970bac87454f4fca0397096365f3c9627cbde8#diff-0dee1f0f5fab61a1beb2855bd256c657ca9426ca4013ec3875bce40ca938f701

File: source/links.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Prebuilt PHP binaries and related packages
 Linux, Windows & MacOS
 **********************
 - `PHP 8.0 <https://jenkins.pmmp.io/job/PHP-8.0-Aggregate/>`_
-- (Windows only) `Microsoft Visual C++ 2017 Redistributable <https://aka.ms/vs/15/release/vc_redist.x64.exe>`_
+- (Windows only) `Microsoft Visual C++ 2019 Redistributable <https://aka.ms/vs/15/release/vc_redist.x64.exe>`_
```
Comment (KygekDev) [inline source/links.rst:26]: I mean you can remove line 26 and tell that the `vc_redist.x64.exe` is included inside the PHP 8.0 artifacts (Windows)

Comment (KygekDev) [reply source/links.rst:26]: Unfortunately I can't give commit suggestion for this line since I don't have write access to your repo or this repo.

Comment (dktapps) [reply source/links.rst:26]: Stop overcomplicating it.

**Pull request merged**